scripts/dagmc_stats.py

```diff
@@ -294,7 +291,7 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
     """
 
     coarseness = []
-    
+
```

Comment:
maybe formatting change should be part of a separate PR ?

(I always to them... and @gonuke ask me for a different PR for formatting changes... this is revenge :) )

Reply:
My text editor automatically strips trailing whitespaces upon saving. Really there should never be trailing whitespace in a PR and we should check for that. I am not making a separate PR for this. Everyone should setup their text editor to always strip trailing whitespace upon saving so we never run into this issue anymore. I will die on that hill.

Reply:
PS- for review purposes, you can ignore whitespace changes when you view the diffs.